styleguide/core/Input.hbs

```diff
@@ -22,8 +22,8 @@
         <div class="{{blockName}}"{{extraAttributes}}>
 
             {{#defineBlockBody}}
-                {{element "input"}}
                 {{element "label"}}
+                {{element "input"}}
```

Comment:
I think having the `input` element before `label` was intentional in the original implementation, to allow for styling via the `+` selector?